Blackstar/BlackstarClient.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Blackstar
+{
+    public class BlackstarClient
```

Comment:
The name `Client` is unambiguous within the library, but if it's used in client code with many other frameworks it might not be clear what kind of client it is. Went with the same convention as `HttpClient`.